doc/en/example/pythoncollection.rst

```diff
@@ -266,3 +269,17 @@ file will be left out:
     collected 0 items
 
     ======================= no tests ran in 0.12 seconds =======================
+
+It's also possible to ignore files based on Unix shell-style wildcards by adding
+patterns to ``collect_ignore_glob``.
```

Comment:
Completely forgot about `collect_ignore`. 😛 

Could you please add `collect_ignore` and `collect_ignore_glob` to the [reference docs](https://docs.pytest.org/en/latest/reference.html)? They should go under the "Special Variables" section.

Reply:
Sure, done.